### PR TITLE
Added spanish (Spain) language

### PR DESCRIPTION
**Pull request by ssergio-ll**

### CODE REVIEW

File: src/i18n/MControlCenter_es.ts [hunk 1]
```diff
@@ -0,0 +1,203 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="es_ES" sourcelanguage="es_ES">
```
Comment (dmitry-s93) [inline src/i18n/MControlCenter_es.ts:3]:
```suggestion
<TS version="2.1" language="es_ES" sourcelanguage="en_US">
```

File: src/i18n/MControlCenter_es.ts [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+        <source>EC Version:</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>EC Build:</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Carga de la batería:</source>
+        <translation type="unfinished"></translation>
```
Comment (dmitry-s93) [inline src/i18n/MControlCenter_es.ts:20]:
```suggestion
        <source>Battery charge:</source>
        <translation>Carga de la batería:</translation>
```

Comment (dmitry-s93) [reply src/i18n/MControlCenter_es.ts:20]: Hello
The translated strings should look like this.

**Comment by dmitry-s93** [comment]
Please update src/i18n/MControlCenter_es.ts as recommended above and I will accept this pull request.

File: src/i18n/MControlCenter_es.ts (hunk 2 above)
Comment (ssergio-ll) [reply src/i18n/MControlCenter_es.ts]: Hi Dmitry.

I have updated my file.

**Pull request merged**